smartconnect/models.py

```diff
@@ -36,28 +53,234 @@ class Config:
         }
 
 
-class Geometry(BaseModel):
-    coordinates: List[float] = Field(..., max_item=2, min_items=2)
+class SMARTRequest(BaseModel):
+    type: str = 'Feature'
+    geometry: Geometry
+    properties: Properties
+
 
+class Waypoint(BaseModel):
+    attachments: List[Any] # test what is contained here
+    conservation_area_uuid: str
+    date: date
+    id: str
+    client_uuid: Optional[str]
+    last_modified: datetime
```

Comment:
I wonder whether we should fix-up timezone during marshaling. How much extra work are we doing in logic to handle timezone issues? Would it save us some headaches to enforce timezone here?

Reply:
timezone handling is all done in one centralized area in the generation of smart api requests. Right now this model is only used to determine the existence of a waypoint by parsing the response to this object. last_modified is not referenced currently. If it were used for updating, its value would already be in format and timezone specific to CA we are updating so might be unnecessary work to convert to standard and convert back

Reply:
Ah perfect. Thanks for the explanation.